wikimedia/wikipedia-ios
# Update Watchlist icon in Account Settings

**Phabricator:** N/A

### Notes
Updates watchlist entry point icon.

### Test Steps
1. Open Account settings
2. Confirm watchlist icon matches Figma
